services/queue/test/queueservice_test.js

```diff
@@ -174,11 +172,9 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
 
     // Poll message again
     message = await testing.poll(async () => {
-      for (let i = 0; i < poll.length; i++) {
-        let messages = await poll[i](1);
-        if (messages.length === 1) {
-          return messages[0];
-        }
+      let messages = await poll(1);
```

Comment:
before `queueService.pendingQueues` returned list of functions, one per each priority, to ask iteratively starting from highest to lowest. 
We no longer need this, since single function can return tasks ordered by priority already